### ISSUE TITLE
Clarify wording on Nextjs troubleshooting page

### ISSUE THREAD

**Issue by vuki656** [getsentry/sentry-docs#5857]
### Core or SDK?

Core Sentry product

### Which part? Which one?

Nextjs

### Description

https://docs.sentry.io/platforms/javascript/guides/nextjs/troubleshooting/

First sentance includes "If you need help solving issues with your Sentry JavaScript SDK integration, you can read the edge cases documented here." but `here` part doesen't have a link attached.

### Suggested Solution

Link to proper doc page

**Comment by imatwawana** [comment]
> Yup, can confirm here. Maybe we can reword this to make it clearer? Say `below` instead of `here`?

That makes sense to me. I can fix that.
